presto-main/src/test/java/com/facebook/presto/sql/planner/TestLogicalPlanner.java

```diff
@@ -1124,6 +1111,25 @@ public void testPushdownSubfieldsIntoConnector()
                 plan -> assertSubfieldPaths(plan, "lineitem_ext", ImmutableMap.of("nested_ints", ImmutableList.of(new SubfieldPath("nested_ints[*][2]")))));
     }
 
+    private void assertSubfieldsInTableScan(Plan plan, String tableName, Map<String, List<SubfieldPath>> expectedSubfields)
+    {
+        TableScanNode tableScan = searchFrom(plan.getRoot())
+                .where(node -> node instanceof TableScanNode && ((TpchTableHandle) ((TableScanNode) node).getTable().getConnectorHandle()).getTableName().equals(tableName))
+                .findOnlyElement();
+
+        Map<Symbol, ColumnHandle> handles = tableScan.getAssignments();
+        Map<ColumnHandle, Symbol> handleToSymbol
```

Comment:
This can be simplified to `Map<ColumnHandle, Symbol> handleToSymbol = ImmutableBiMap.copyOf(tableScan.getAssignments()).inverse();`

Reply:
Yup that's better. Thanks.